src/lib/loggers/morganMiddleware.js

```diff
@@ -0,0 +1,43 @@
+import morgan from 'morgan';
+
+import { Logger } from './logger';
+
+// not sure how to convert typescript below to javascript, so I added the object
+// directly into the variable below
+
+// Override the stream method by telling
+// Morgan to use our custom logger instead of the console.log.
+// const stream: StreamOptions = {
+//   // Use the http severity
+//   write: (message) => Logger.http(message),
+// };
+
+// Skip all the Morgan http log if the
+// application is not running in development mode.
+// This method is not really needed here since
+// we already told to the logger that it should print
+// only warning and error messages in production.
+const skip = () => {
```

Comment:
### Nitpick

For the morgan middleware definition I'l d say have in the middlewares directory, but the configuration of it can be in the `lib` directory